### ISSUE TITLE
Sugestions

### ISSUE THREAD

**Issue by painoob** [niutech/splitbrowser#5]
This project is awesome, thank you in first place.

I intend to use it as a dashboard wall, to monitore multiple tools. It would be usefull if you add some features:

- Fullscreen support (for the whole window, not tiles)
- Hide tab bar, address bar and command buttons( as forward, back and refresh)

Example:

![Dashboard](https://media.licdn.com/dms/image/C5612AQH0gYw0UlEJlg/article-cover_image-shrink_600_2000/0/1521472625644?e=2147483647&v=beta&t=m3aEK5eqBDo7omnI39Br5g-8xN0sbScsZEQ4JlMzCAs)

**Comment by niutech** [comment]
You're welcome. Full screen mode is possible using `QMainWindow::showFullScreen()` - I'll try to implement it.

**Comment by Khaled270** [comment]
Absolutely, I couldn't agree more! This project is truly impressive, and I've been using it in the same way. The proposed changes sound fantastic - full-screen support and the option to hide unnecessary elements would be great improvements. I'm really looking forward to seeing these changes implemented!

**Comment by Khaled270** [comment]
: (

**Comment by ioc2e3** [comment]
> 這個項目太棒了，首先謝謝你。

Unfortunately ...

Still no update so far ...

.
